pyengine/katana.py

```diff
@@ -82,7 +93,7 @@ def quotetrimmer(side,limitprice,book):
 
     for quote in book.copy():
         quoteprice = book[quote][0]
-        if compareprice(side,limitprice,quoteprice):
+        if compareprice(side,limitprice,quoteprice,1,0):
```

Comment:
if you want this behavior, default q1=1 and q2=0 in the compareprice function itself dont add it here where its not needed

Reply:
done